extensions/functions_arithmetic_decimal.yaml

```diff
@@ -102,14 +102,96 @@ scalar_functions:
           scale_after_borrow = max(init_scale - delta, min_scale)
           scale = init_prec > 38 ? scale_after_borrow : init_scale
           DECIMAL<prec, scale>
-  - 
+  -
     name: "abs"
     description: Calculate the absolute value of the argument.
     impls:
       - args:
           - name: x
             value: decimal<P,S>
         return: decimal<P,S>
+  -
+    name: "power"
+    description: "Take the power with x as the base and y as exponent."
+    impls:
+      - args:
+          - name: x
+            value: "DECIMAL<P,S>"
```

Comment:
I don't think these data types are right. This says that both arguments must have the same precision and scale. I think this and all the ones below are not right.

Reply:
Fixed power function. I have removed all other functions, I will add them in separate PR